modules/candidate_parameters/php/candidatequeryengine.class.inc

```diff
@@ -74,7 +74,7 @@ class CandidateQueryEngine extends \LORIS\Data\Query\SQLQueryEngine
                     "Sex",
                     "Candidate's biological sex",
                     $candscope,
-                    new \LORIS\Data\Types\Enumeration('Male', 'Female', 'Other'),
+                    new \LORIS\Data\Types\StringType(255),
```

Comment:
The domain is restricted and we still want the options to be a dropdown. This should be an enum of `\Utility::getSexList()...`, not a string.